packages/transform-metering/src/meter.js

```diff
@@ -8,6 +8,20 @@ const ObjectConstructor = Object;
 const bigIntWord = 2n ** 64n;
 const bigIntZero = 0n;
 
+/*
+ * Getting the properties of a SES-shim module exports namespace object throws
+ * an error if inspected before the module initializes.
+ * Allowing this case is necessary for the compartment mapper to be hosted by
+ * the compartment mapper under metering.
+ */
+const maybeGetOwnPropertyDescriptors = object => {
+  try {
+    return getOwnPropertyDescriptors(object);
+  } catch (_error) {
+    return {};
+  }
+};
+
```

Comment:
This change is necessary for making Zip files work on Node.js with metering. We might not be exercising this anymore, but I include the change for completeness. cc @michaelfig